builtin/common/after.lua

```diff
@@ -117,6 +118,12 @@ local time_next = math.huge
 core.register_globalstep(function(dtime)
 	time = time + dtime
 
+	for _, job in pairs(job_map) do
+		if job.elapsed_time then
+			job.elapsed_time = job.elapsed_time + dtime
+		end
+	end
```

Comment:
There has been put work into core.after specifically to not iterate through all jobs at each step, for performance reason.

You could store the time when the job was inserted, and in `get_elapsed_time` compare it with the current game time.

Reply:
Done